app/src/main/java/it/quip/android/QuipitApplication.java

```diff
@@ -17,7 +17,7 @@ public void onCreate() {
         ParseInstallation.getCurrentInstallation().saveInBackground();
     }
 
-    private String stringRes(int resId) {
+    public String stringRes(int resId) {
```

Comment:
I can move this back to private, but i was trying to getRes on for having
strings instead of hardcoded constants

On Sun, Oct 18, 2015 at 7:28 PM, Hasham Ali <notifications@github.com>
wrote:

> In app/src/main/java/it/quip/android/QuipitApplication.java
> <https://github.com/quipit/quipit-android/pull/7#discussion_r42332434>:
>
> > @@ -17,7 +17,7 @@ public void onCreate() {
> >          ParseInstallation.getCurrentInstallation().saveInBackground();
> >      }
> >
> > -    private String stringRes(int resId) {
> > +    public String stringRes(int resId) {
>
> what is this for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/quipit/quipit-android/pull/7/files#r42332434>.
>